packages/prisma/schema.prisma

```diff
@@ -287,7 +289,7 @@ enum BookingStatus {
 model Booking {
   id                    Int                  @id @default(autoincrement())
   uid                   String               @unique
-  user                  User?                @relation(fields: [userId], references: [id])
+  user                  User[]
```

Comment:
Some observations:
- Are we trying to keep track of attendees here?
- How do we differentiate between an owner and attendee?
- What does it means that a Booking has a ´user´?